tools/stern/src/Stern/Intra.hs

```diff
@@ -404,8 +404,7 @@ getUserBindingTeam u = do
   pure $
     listToMaybe $
       fmap (view teamId) $
-        filter ((== Binding) . view teamBinding) $
-          teams ^. teamListTeams
+        teams ^. teamListTeams
```

Comment:
This makes me think that perhaps you should filter the galley cql queries for the `binding` field.  We don't *expect* there to be any non-binding teams in the database, but for robustness we might as well make sure we don't suddenly treat find them and treat them as binding teams.

Reply:
https://github.com/wireapp/wire-server/pull/2514/commits/fbee436ab2f362f01c8d1604ed76778b87bea3a2

did i forget anything?